controllers/velero.go

```diff
@@ -425,7 +425,7 @@ func (r *DPAReconciler) appendPluginSpecificSpecs(veleroDeployment *appsv1.Deplo
 					ImagePullPolicy:          imagePullPolicy,
 					Resources:                init_container_resources,
 					TerminationMessagePath:   "/dev/termination-log",
-					TerminationMessagePolicy: "File",
+					TerminationMessagePolicy: corev1.TerminationMessageFallbackToLogsOnError,
```

Comment:
here would not be better to use `setContainerDefaults` function from controllers/common.go?

Reply:
We use the `setContainerDefaults` function for velero and node-agent containers, here are we setting values for plugin init containers. so I think current way of things is fine for now.